pkg/koordlet/statesinformer/states_noderesourcetopology.go

```diff
@@ -94,34 +105,89 @@ func (s *nodeTopoInformer) Setup(ctx *pluginOption, state *pluginState) {
 		klog.Fatalf("pods informer format error")
 	}
 	s.podsInformer = podsInformer
+
+	s.NodeResourceTopologyInformer.AddEventHandler(cache.ResourceEventHandlerFuncs{
```

Comment:
a special case: if the feature-gate `NodeTopologyReport` is disabled, we do not need to start the informer since the NRT object is only updated in-memory.